mne/stats/cluster_level.py

```diff
@@ -1261,6 +1286,18 @@ def spatio_temporal_cluster_1samp_test(
     H0 : array, shape (n_permutations,)
         Max cluster level stats observed under permutation.
 
+    Notes
+    -----
+    For computing a ``threshold`` based on a p-value, use the conversion
+    from :meth:`scipy.stats.t.ppf`::
+
+        pval = 0.001  # arbitrary
+        df = n_observations - 1  # degrees of freedom for the test
+        thresh = scipy.stats.t.ppf(1 - pval /  2, df)  # two-tailed
+
+    For a one-tailed test (``tail=1``), don't divide the p-value by 2.
+    For testing the lower tail (``tail=-1``), don't subtract pval from 1.
```

Comment:
not DRY.  If this info needs to be in the docstring (I'm inclined to agree that it does) and it needs to show up in multiple places, let's add it to the docdict.

Reply:
agreed, see: a9197fde6f277e8758a288389c1ba727fb781720